app/src/main/res/layout/search_result_item.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="?android:attr/listPreferredItemHeight"
+    android:padding="6dip" >
```

Comment:
Use `dp `instead of `dip` it's the same thing but `dp `is used in our other files

Reply:
Done